src/main/java/com/softserveinc/ita/shladkyi/GoogleHomePage.java

```diff
@@ -16,25 +17,21 @@ public GoogleHomePage open() {
     }
 
     public GoogleSearchResultPage searchFor(String searchTerm) {
-        TestRunner.getDriver()
-                .findElement(By.cssSelector("[class='gLFyf gsfi']"))
-                .sendKeys(searchTerm, Keys.ENTER);
+        WebElementUtil.setValueForElement("//input[@class='gLFyf gsfi']", searchTerm + Keys.ENTER);
         return new GoogleSearchResultPage();
     }
 
     public GoogleSettingsPage openSettingsPage() {
-        TestRunner.getDriver().findElement(By.id("Mses6b")).click();
-        WebElement searchSettingsButton = TestRunner.getDriver().findElement(By.xpath("//a[@class='EzVRq']"));
-        TestRunner.getWait().until(ExpectedConditions.elementToBeClickable(searchSettingsButton));
-        searchSettingsButton.click();
+        WebElementUtil.clickElement("//button[@id='Mses6b']");
+        WebElementUtil.clickElement("//a[@class='EzVRq']");
         return new GoogleSettingsPage();
     }
 
     public String getSearchButtonText() {
-        return TestRunner.getDriver().findElement(By.cssSelector("[name='btnK']")).getAttribute("value");
+        return TestRunner.getDriver().findElement(By.xpath("//input[@name='btnK']")).getAttribute("value");
```

Comment:
use wait

Reply:
When i use wait on this element I always have TimeoutException. I cant find why this works like this.